### PR TITLE
refactor: response pack link to open in new tab(CMC-422)

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
[CMC-422](https://tools.hmcts.net/jira/browse/CMC-422)

### Change description ###
This PR contains the change to open the response pack link in new tab on confirmation screen. Also we are making the link configurable in application yaml.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ckalista — DISMISSED** [review]

**Review by dharmendrak — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java [hunk 1]
```diff
@@ -67,14 +70,13 @@ void shouldReturnExpectedSubmittedCallbackResponseObject() {
...
         String documentLink = "https://www.google.com";
-        String responsePackLink = "https://formfinder.hmctsformfinder.justice.gov.uk/n9-eng.pdf";
         LocalDateTime serviceDeadline = LocalDate.now().plusDays(112).atTime(23, 59);
         String formattedServiceDeadline = formatLocalDateTime(serviceDeadline, DATE_TIME_AT);
 
         String body = format(
             "<br />Follow these steps to serve a claim:"
                 + "\n* [Download the sealed claim form](%s) (PDF, 123KB)"
-                + "\n* Send the form, particulars of claim and [a response pack](%s) (PDF, 266 KB) "
+                + "\n* Send the form, particulars of claim and [a response pack](%s target=\"_blank\") (PDF, 266 KB) "
```
Comment (swalker125) [inline src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java]: Does this actually work?

**Review by swalker125 — COMMENTED** [review]
I would say a refactor more than a fix! 😁

File: src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java (hunk 1 above)
Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java]: yes it does at least in chrome. Not tested in all browsers.

Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java]: It doesn't work in incognito mode of chrome but works in normal mode of chrome.

Comment (sabahirfan) [reply src/test/java/uk/gov/hmcts/reform/ucmc/handler/CreateClaimCallbackHandlerTest.java]: Replacing this with href ta as this doesn't work with Safari either.

**Review by swalker125 — APPROVED** [review]

**Review by ckalista — APPROVED** [review]

**Pull request merged**